megamek/src/megamek/client/ui/swing/GUIPreferences.java

```diff
@@ -347,6 +349,7 @@ protected GUIPreferences() {
         store.setDefault(GAME_SUMMARY_MINI_MAP, false);
         store.setDefault(ENTITY_OWNER_LABEL_COLOR, true);
         store.setDefault(UNIT_LABEL_BORDER, true);
+        store.setDefault(UNIT_LABEL_STYLE, "NICKNAME");
```

Comment:
Can we please use the enum .name method instead? Helps prevent issues with name changes (I've made this mistake in MekHQ before)